R/distanceMatrix.R

```diff
@@ -4,30 +4,29 @@
 #' @param t An optional list of the same size as \code{q} containing evaluation
 #'   grids for each QTS.
 #' @param labels A character vector specifying labels for each QTS.
+#' @param step_pattern The choice for the step pattern of the warping path (see
+#'  \code{\link[dtw]{stepPattern}} for more information)
```

Comment:
Please end documentation with a proper period at the end of the sentence and specify what is the choice by default with a specific sentence like "Defaults to `dtw::symmetric2`" and explain briefly what this default does.